README.md

```diff
@@ -58,7 +58,7 @@ Migrate the database
 	
 Run the tests
 
-    pytest -v --cov django_basin3d  tests 
+    pytest -v --cov django_basin3d example-django/tests 
```

Comment:
The manage.py file is also in example-django directory. Needs to be changed on line 57, 66, and 71.